Change usage to use throughout spec text

wip-abramson opened the pull request:
This P.R. addresses issue #1555 

Just checking, I should only be changing the base index.html right? 

Also, as I was going through this I noticed the spec still has 32 uses of the word usage. I believe in the examples.

For example: "Usage of the credentialSubject property"

Should we also be changing these to Use?


<!--
    This comment and the below content is programmatically generated.
    You may add a comma-separated list of anchors you'd like a
    direct link to below (e.g. #idl-serializers, #idl-sequence):

    Don't remove this comment or modify anything below this line.
    If you don't want a preview generated for this pull request,
    just replace the whole of this comment's content by "no preview"
    and remove what's below.
-->
***
<a href="https://pr-preview.s3.amazonaws.com/wip-abramson/vc-data-model/pull/1561.html" title="Last updated on Oct 3, 2024, 4:04 PM UTC (d656c37)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/1561/062e4b3...wip-abramson:d656c37.html" title="Last updated on Oct 3, 2024, 4:04 PM UTC (d656c37)">Diff</a>

TallTed commented:
> Should we also be changing these to Use?

Yes, "use" is fully preferred to "usage". Depending on the specific phrasing, sometimes this change requires other minor rewording.

TallTed commented:
I've just reviewed every occurrence of `usage`. I think all can and should be replaced by `use`, though a few instances will require rephrasing of the neighboring or containing phrase. I can produce a PR for this, after this one (#1561) is merged; or you could change them on your branch, within this PR, and I can then suggest the rephrasing (which I cannot suggest until the changes of `usage` makes the neighboring lines available for change requests).

wip-abramson commented:
@TallTed I have replaced usage with use throughout the spec. I have mostly left any additional rephrasing required to you as suggested.

I renamed the P.R. to more accurately reflect the changes

TallTed commented on index.html:
```diff
@@ -1733,7 +1733,7 @@ <h3>Names and Descriptions</h3>
 
         <pre class="example nohighlight"
-          title="Usage of the name and description property">
+          title="Use of the name and description property">
```
```suggestion
          title="Use of the name and description properties">
```

TallTed commented on index.html:
```diff
@@ -1836,7 +1836,7 @@ <h3>Issuer</h3>
         </dl>
 
-        <pre class="example nohighlight vc" title="Usage of issuer property"
+        <pre class="example nohighlight vc" title="Use of issuer property"
```
```suggestion
        <pre class="example nohighlight vc" title="Use of the issuer property"
```

TallTed commented on index.html:
```diff
@@ -1863,7 +1863,7 @@ <h3>Issuer</h3>
 
         <pre class="example nohighlight vc"
-          title="Usage of issuer expanded property"
+          title="Use of issuer expanded property"
```
```suggestion
          title="Expanded use of the issuer property"
```

TallTed reviewed (changes requested):
Editorial. Small grammar fixes.

wip-abramson commented on index.html:
```diff
@@ -5624,7 +5624,7 @@ <h3>Usage Patterns</h3>
 
         <p>
-Privacy violations of [=verifiable credential=] usage occur when unintended or
+Privacy violations of [=verifiable credential=] use occur when unintended or
 unexpected correlation arises from the presentation of those
 [=verifiable credentials=].
```
```suggestion
Privacy violations related to the use of [=verifiable credentials=] occur when
unintended or unexpected correlation arises from the presentation of those
[=verifiable credentials=].
```

Is this better?

TallTed replied:
I've updated my suggestion to match @wip-abramson's.

TallTed commented on index.html:
```diff
@@ -4274,7 +4274,7 @@ <h3>Notable JSON-LD Features</h3>
+In order to increase interoperability, this specification restricts the use of
 JSON-LD representations of the data model. JSON-LD <a
 href="https://www.w3.org/TR/json-ld/#compacted-document-form">compacted document
 form</a> MUST be utilized for all representations of the data model using the
```
```suggestion
form</a> MUST be used for all representations of the data model using the
```

TallTed commented on index.html:
```diff
@@ -6460,7 +6460,7 @@ <h2>Validation</h2>
 [=verifiable presentations=], readers might be curious about how the
 information in this data model is expected to be utilized by [=verifiers=]
```
```suggestion
information in this data model is expected to be used by [=verifiers=]
```

TallTed reviewed (commented):
"utiliz{ed,ation,ing}" should also all become "us{ed,e,ing}", like these two.

wip-abramson commented:
@TallTed I believe this addresses all uses of utiliz{e,es,ing,ed} throughout the spec.

msporny commented:
There are merge conflicts with the huge security considerations grammar fix PR. Please fix the merge conflicts @wip-abramson and we can get this merged.

wip-abramson commented:
@msporny should be good to go

TallTed commented on index.html:
```diff
@@ -5555,12 +5554,11 @@ <h3>Patterns of Use</h3>
+The centralized server can correlate the [=credential|credential's=] use across
+interactions. For example, if a [=credential=] is used to prove age in this
+manner, the centralized service could know everywhere that [=credential=] was
+presented (all liquor stores, bars, adult stores, lottery sellers, and so on).
```
```suggestion
The centralized server can correlate uses of the [=credential=] across
interactions. For example, if a [=credential=] is used to prove age in this
manner, the centralized service could know everywhere that [=credential=] was
presented (all liquor stores, bars, adult stores, lottery sellers, and so on).
```

TallTed replied:
Redone in #1569

TallTed commented on index.html:
```diff
@@ -5934,20 +5931,21 @@ <h3>Content Integrity Protection</h3>
-[=verifiable credential=], it is strongly suggested for external resources
-that could result in a security issue if the external content changes.
+Section [[[#integrity-of-related-resources]]] of this specification provides an
+optional mechanism for ensuring content integrity of external
```
```suggestion
optional mechanism for ensuring the integrity of the content of external
```

TallTed replied:
Redone in #1569

TallTed reviewed (changes requested):
Small, editorial

msporny commented:
@wip-abramson can you take care of @TallTed's suggestions so we can get this merged? https://github.com/w3c/vc-data-model/pull/1561#pullrequestreview-2346395835

I would, but it's not showing up in my source view for some reason... wonder if Ted submitted his changes based on some different/disconnected commit hash?

TallTed commented:
Very odd. I made my suggestions on the `/files` view, but now that doesn't obviously include these paragraphs, and digging deeper, they're in the "untouchable" blue section of the document. Redone in #1569.

msporny commented:
Editorial, multiple reviews, changes requested and made, no objections, merging.
